Tests/FoundationEssentialsTests/GregorianCalendarRecurrenceRuleTests.swift

```diff
@@ -0,0 +1,671 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if canImport(TestSupport)
+import TestSupport
+#endif
+
+#if FOUNDATION_FRAMEWORK
+@testable import Foundation
+#else
+@testable import FoundationInternationalization
+@testable import FoundationEssentials
+#endif // FOUNDATION_FRAMEWORK
+
+@available(FoundationPreview 0.4, *)
+final class CalendarRecurrenceRuleTests: XCTestCase {
+    /// A Gregorian calendar with a time zone set to California
+    var gregorian: Calendar = {
+        var gregorian = Calendar(identifier: .gregorian)
+        gregorian.timeZone = .init(identifier: "US/Pacific")!
```

Comment:
Naming the time zone "US/Pacific" won't work in an Essentials-only test, which may not have time zone data. You can use a fixed GMT-8 if you like; or, if your test requires checking daylight saving transitions, move this up to the internationalization tests.

I don't have an issue with Essentials functionality being tested in the Internationalization tests, if that gives us better test coverage. They really all run together anyway.

Reply:
Moved tests that depend on this timezone to Internalization tests, all others are in GMT now.